zerver/lib/onboarding.py

```diff
@@ -314,6 +314,19 @@ def send_initial_realm_messages(realm: Realm) -> None:
             start_topic_help_url="/help/starting-a-new-topic",
         )
 
+        content_of_zulip_update_announcements_topic_name = (
+            _("This topic will receive automated updates about new Zulip features.") + "\n"
+            "\n"
+            + _(
+                "If you like, you can [configure]({settings_url}) a different stream "
+                "for these updates (and [move]({move_content_help_url}) the initial updates there), "
+                "or [turn this feature off]({settings_url}) altogether."
+            )
```

Comment:
@prakhar1144 so I think this message might be what we want as "level 1" in the system -- that way, existing installations that upgrade to Zulip 9.0 will get this message explaining what the topic is about. We still want it to be also send during realm creation, such that the first message in the "Zulip updates" topics should always be this one explaining what the topic is about.

I guess we also want it to be sent for new organizations, though; perhaps you can figure out a way to not have the content be duplicated?

And then probably @alya will want to rewrite the content to be designed for both of those contexts.

Reply:
> I guess we also want it to be sent for new organizations

This was our plan as discussed on CZO:
> New realms are initialized with the current/latest zulip_updates_level -- i.e. so newly created realms won't get a bunch of old notices (though we may want to add an onboarding message sent to the default stream location for this).

So, I think, new realms should only get the onboarding message & then new updates onwards (no duplication). Right? -- or we want to send all the notices ?

Reply:
New realms created get the onboarding message.

Existing realms get the same message as a part of zulip update announcements.

New realms will be initialized to the latest realm, so they won't get the first announcement -- i.e. no duplication.

Reply:
> Existing realms get the same message as a part of zulip update announcements.

I think they only get it in a direct message to administrators, though -- we probably want something that goes to the actual topic.

Reply:
Ahh, OK, I see you have the same text in both places. I think basically this is ready for me to do a wording edit pass with @alya and then we'll probably be good to go.